pkg/tbtc/heartbeat.go

```diff
@@ -2,27 +2,43 @@ package tbtc
 
 import (
 	"context"
+	"encoding/hex"
 	"fmt"
+	"math/big"
+	"sync"
+
 	"github.com/ipfs/go-log/v2"
 	"github.com/keep-network/keep-core/pkg/bitcoin"
+	"github.com/keep-network/keep-core/pkg/protocol/group"
 	"github.com/keep-network/keep-core/pkg/tecdsa"
-	"math/big"
 )
 
 const (
-	// heartbeatProposalValidityBlocks determines the wallet heartbeat proposal
-	// validity time expressed in blocks. In other words, this is the worst-case
-	// time for a wallet heartbeat during which the wallet is busy and cannot
-	// take another actions. The value of 300 blocks is roughly 1 hour, assuming
-	// 12 seconds per block.
-	heartbeatProposalValidityBlocks = 300
+	// heartbeatTotalProposalValidityBlocks determines the total wallet
+	// heartbeat proposal validity time expressed in blocks. In other words,
+	// this is the worst-case time for a wallet heartbeat during which the
+	// wallet is busy and cannot take another actions. It includes the total
+	// duration need to perform both both signing the heartbeat message and
+	// optionally notifying about operator inactivity if the heartbeat failed.
+	// The value of 600 blocks is roughly 2 hours, assuming 12 seconds per block.
+	heartbeatTotalProposalValidityBlocks = 600
+	// heartbeatInactivityNotificationValidityBlocks determines the duration
+	// that needs to be preserved for the optional notification about operator
+	// inactivity that follows a failed heartbeat signing.
+	heartbeatInactivityNotificationValidityBlocks = 300
```

Comment:
Let's be consistent:
```suggestion
	heartbeatInactivityClaimValidityBlocks = 300
```

Reply:
Done in 7137d4f01fb7a99807664d61f4d62b8407d3dee1.